Formula/unxip.rb

```diff
@@ -0,0 +1,24 @@
+class Unxip < Formula
+  desc "Fast Xcode unarchiver"
+  homepage "https://github.com/saagarjha/unxip"
+  url "https://github.com/saagarjha/unxip.git", tag: "v1.1.3", revision: "b3ce1ab4728d4173390c97eddd28e821ad9e2974"
+  license "LGPL-3.0-only"
+  head "https://github.com/saagarjha/unxip.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on macos: :monterey
+  uses_from_macos "swift"
+
+  def install
+    system "swiftc", "-O", "-parse-as-library", "unxip.swift"
+    bin.install "unxip"
+  end
+
+  test do
+    system "#{bin}/unxip", "--help"
```

Comment:
Since this needs a whole Xcode .xip in order to test functionality, this is about the best we can do today as far as a `test`.

Reply:
How about trying to open a txt file and seeing if it fails correctly?

Reply:
Thanks for the suggestion! It looks like the best I could do for now as a test, is to create a dummy file that's close-ish to a .xip, attempt to expand to a nonexistent directory and test the exit status and error message.

`xip` built-in can actually create xip archives, but they require a signing identity, and only Apple identities are trusted on the system anyway.